### PR TITLE
Disable schema validation

### PR DESCRIPTION
**Pull request by andyatmethods**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -19,9 +19,9 @@ const { getErrorHandler, ejsErrorPages } = require('login.dfe.express-error-hand
 const registerRoutes = require('./routes');
 const setCorrelationId = require('express-mw-correlation-id');
 const { setUserContext } = require('./infrastructure/utils');
-const configSchema = require('./infrastructure/config/schema');
+//const configSchema = require('./infrastructure/config/schema');
```
Comment (AMcCartneyMTH) [inline src/index.js:22]: Could we remove the unused imports, please? These can be re-added if required.

File: src/index.js [hunk 2]
```diff
@@ -19,9 +19,9 @@ const { getErrorHandler, ejsErrorPages } = require('login.dfe.express-error-hand
...
 
-configSchema.validate();
+//configSchema.validate();
```
Comment (AMcCartneyMTH) [inline src/index.js:24]: Could we remove the unused imports, please? These can be re-added if required.

**Review by AMcCartneyMTH — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (andyatmethods) [reply src/index.js]: removed rather than commenting out - will add back if this resolves the issue.

File: src/index.js (hunk 2 above)
Comment (andyatmethods) [reply src/index.js]: removed rather than commenting out - will add back if this resolves the issue.

**Review by ageerlenin — APPROVED** [review]

**Review by AMcCartneyMTH — APPROVED** [review]

**Pull request merged**